docs/_includes/header_custom.html

```diff
@@ -2,4 +2,8 @@
 #main-content p {
     text-align: justify;
 }
+
+.language-mermaid .label {
+    text-transform: inherit;
+}
```

Comment:
Context:

### Before

upper case mermaid labels

![image](https://user-images.githubusercontent.com/60357436/214560132-9771f84e-3c94-4802-b62d-a30fd477c111.png)

### After

normal case mermaid labels

![image](https://user-images.githubusercontent.com/60357436/214560160-4ea4f407-54b1-43a0-8b01-ae529dbddda1.png)

We should get that fixed [upstream](https://github.com/just-the-docs/just-the-docs/blob/3d1f926a6821713598104aca6dbb28debca2a02f/_sass/code.scss#L204) - but not a blocker for now